bin/build-all.sh

```diff
@@ -12,23 +12,24 @@ targets=(
 	6.9.5     # 48
 	7.5.0     # 51
 )
+
+if [[ $NODE == "" ]]; then
+	NODE=`which node`
+fi
+
 cwd=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 node_pre_gyp="$(dirname "$cwd")/node_modules/node-pre-gyp/bin/node-pre-gyp"
 args='rebuild'
 
 if [[ $npm_lifecycle_event == "prepublish" ]]; then
-	is_publish=`$NODE -e "console.log(JSON.parse(process.env.npm_config_argv).cooked.indexOf('publish') !== -1);"`
+	is_publish=`$NODE -e "console.log(JSON.parse('$npm_config_argv').cooked.indexOf('publish') !== -1);"`
```

Comment:
I actually found that the env variable didn't end up "propagating down" to the node process we're spawning here. So $npm_config_argv was defined when bin/build-all.sh was executed, but wasn't set on the `node -e` command we fire here (at least on my machine)